packages/bp/src/sdk/rewire.ts

```diff
@@ -10,9 +10,7 @@ const nativeBindingsPaths: string[] = []
 
 const nativeExBaseFolder =
   (process.core_env.NATIVE_EXTENSIONS_DIR && syspath.resolve(process.env.NATIVE_EXTENSIONS_DIR!)) ||
```

Comment:
Why keep the env var here?

Reply:
Retrocompatibility. I know bindings didn't work on all systems at all time, maybe some users added their own bindings somewhere. Otherwise, I have no other reasons